code/modules/surgery/organs/augments_arms.dm

```diff
@@ -167,6 +168,14 @@
 		span_hear("You hear a short mechanical noise."))
 	playsound(get_turf(owner), extend_sound, 50, TRUE)
 
+	if(length(items_list) > 1)
+		RegisterSignal(active_item, list(COMSIG_ITEM_ATTACK_SELF, COMSIG_ITEM_ATTACK_SELF_SECONDARY, PROC_REF(swap_tools))) // secondary for welders
```

Comment:
couldn't it just be secondary for everything?

Reply:
No, as primary is easier to use because you can press the 'use in hand' hotkey, which secondary does not currently have, and even if it did, most people wouldn't know about it.